src/Agent.Listener/CommandSettings.cs

```diff
@@ -163,6 +164,22 @@ public bool GetRunAsService()
                 defaultValue: false);
         }
 
+        public bool GetEnableAutoLogon()
+        {
+            return TestFlagOrPrompt(
+                name: Constants.Agent.CommandLine.Flags.EnableAutoLogon,
+                description: StringUtil.Loc("EnableAutoLogon"),
+                defaultValue: false);
+        }
+
+        public bool GetRestartNow()
```

Comment:
discussed it with Prachi and true is something we want to go with. Have discussed this over mail as well. Resolving the comment.

---
In reply to: [118593605](https://github.com/Microsoft/vsts-agent/pull/985#discussion_r118593605) [](ancestors = 118593605)